cmd/process-agent/api/server.go

```diff
@@ -40,16 +40,8 @@ func SetupAPIServerHandlers(deps APIServerDeps, r *mux.Router) {
 	r.HandleFunc("/config", deps.Settings.GetFullConfig("process_config")).Methods("GET")
 	r.HandleFunc("/config/all", deps.Settings.GetFullConfig("")).Methods("GET") // Get all fields from process-agent Config object
 	r.HandleFunc("/config/list-runtime", deps.Settings.ListConfigurable).Methods("GET")
-	r.HandleFunc("/config/{setting}", func(w http.ResponseWriter, r *http.Request) {
-		vars := mux.Vars(r)
-		setting := vars["setting"]
-		deps.Settings.GetValue(setting, w, r)
-	}).Methods("GET")
-	r.HandleFunc("/config/{setting}", func(w http.ResponseWriter, r *http.Request) {
-		vars := mux.Vars(r)
-		setting := vars["setting"]
-		deps.Settings.SetValue(setting, w, r)
-	}).Methods("POST")
+	r.HandleFunc("/config/{setting}", deps.Settings.GetValue).Methods("GET")
+	r.HandleFunc("/config/{setting}", deps.Settings.SetValue).Methods("GET")
```

Comment:
Should this be `POST`?

Reply:
yes 🤦‍♂️ thank you for catching this

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/25440/commits/72800a58f5c5f3c0179d23c5d614fc9da65d882c